recipes/czml3/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "czml3" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d97f0b503de13faeb5e8dab7165279b2f1d948286b60378afe6e8dcef5a70df3
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True
```

Comment:
Why is this here?
```suggestion
```

Reply:
Oops, leftover from a wrong version of applying the version specifiers